Docs/General/ElectronicsCrashCourse.md

```diff
@@ -197,11 +232,11 @@ The CANivore gets around some hardware limitations of the Rio to have extra band
 Encoders are devices that measure rotation.
 We use them to measure the angles of our swerve modules, the speed of our wheels, the position of our elevator, and much more.
 Modern motors have encoders built in.
-However these encoders are relative encoders.
-Relative encoders are set to 0 when the robot is turned on, and track the change in position from there.
-If all we care about measuring is velocity this is fine, but if we wanted to know how far our elevator is extended or what angle our swerve module is at we would need to align them to some known position whenever we started up our robot.
-Instead we can use absolute encoders to measure the absolute position of a mechanism.
```

Comment:
i feel like we should keep this

Reply:
the main thing is that they arent actually relative, and we likely wont encounter relative encoders anymore